kubernetes/test-infra/eks/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     kubernetes = {
       source  = "hashicorp/kubernetes"
-      version = "1.13"
+      version = "9.9.9"
```

Comment:
Is this version change on purpose or a left-over from testing?

Reply:
It's a left-over. Thanks for catching this!

Reply:
oh oops, actually, this is in the test-infra dir, so it's on purpose. This PR should only modify `_examples`, which should not have 9.9.9 in them.